docs/reference/edot-collector/custom-collector.md

```diff
@@ -31,10 +31,6 @@ Refer to the following sections to complete these steps.
 
 Install `ocb` using the command that aligns with your system from the [OpenTelemetry building a custom Collector documentation](https://opentelemetry.io/docs/collector/custom-collector/#step-1---install-the-builder).
 
-:::{warning}
-Make sure to install version 0.120.1 of the OpenTelemetry Collector Builder.
```

Comment:
instead of removing that should we maybe just change it to call out that that a specific version of OCB needs to be installed (i.e. the one from {{ otel_col_version }} )